docs/review_checklist.md

```diff
@@ -30,7 +30,7 @@ Below is an example of the review checklist for the [Yellowbrick JOSS submission
 
 ### Documentation
 
-- **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
+- **A statement of need:** Do the authors have a Statement of Need that clearly states what problems the software is designed to solve and who the target audience is?
```

Comment:
Should it be yet more specific as in "Do the authors have a section entitled Statement of Need..." since that is what exactly is checked for?